stdlib/3/email/generator.pyi

```diff
@@ -25,4 +25,4 @@ class BytesGenerator:
 class DecodedGenerator(Generator):
     # TODO `fmt` is positional
     def __init__(self, outfp: TextIO, mangle_from_: bool = ...,
-                 maxheaderlen: int = ..., *, fmt: Optional[str]) -> None: ...
+                 maxheaderlen: int = ..., *, fmt: Optional[str] = ...) -> None: ...
```

Comment:
Could you also remove the asterisk and the TODO comment above? I believe they were only added because without a default there was a syntax error here. But since you fixed that they are not needed anymore.